app/src/main/AndroidManifest.xml

```diff
@@ -78,6 +78,7 @@
             android:configChanges="keyboard|keyboardHidden|mcc|mnc|orientation|screenSize|layoutDirection|smallestScreenSize|screenLayout"
             android:launchMode="singleTask"
             android:resizeableActivity="true"
+            android:exported="true"
```

Comment:
Why do we need to make this activity exported? It means that third-party apps can now launch our apps in ways that we may not be able to control.

We usually want `IntentReceiverActivity` to be the only exposed surface area.

Reply:
Now it's required to indicate the exported behavior. But before it looks like this was the entry point for other deep links? The other deep link were not public?  https://github.com/mozilla-mobile/fenix/pull/23487/files/898c18d41471c3d9075f5642bd8bb138ca2a1f29#diff-7fa6aef292187a049f7a4d6060d8df3ba212d838789c78940bd363344b1c38cdR101-R123

Reply:
Resolved in latest commit